src/app/components/attestation/attestation-create.jsx

```diff
@@ -137,9 +174,11 @@ function ChplAttestationCreate(props) {
 
   return (
     <ThemeProvider theme={theme}>
-      <Typography variant="h1">
-        Submit Attestation
-      </Typography>
+      <div className={classes.pageHeader}>
```

Comment:
The styles in this class don't seem to do anything, except for the padding one. Do we need the other three?